app/src/main/java/com/daniel/card_game_android/Player.java

```diff
@@ -0,0 +1,65 @@
+package com.daniel.card_game_android;
+
+import android.content.Intent;
+public class Player {
```

Comment:
Looks good i think we dont need to send the intent. we need to think on another way.